src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Samples.cshtml.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using ApiView;
+using APIView;
+using APIViewWeb.Models;
+using APIViewWeb.Repositories;
+using Markdig.Helpers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json.Linq;
+
+namespace APIViewWeb.Pages.Assemblies
+{
+    public class UsageSamplePageModel : PageModel
+    {
+        private readonly UsageSampleManager _samplesManager;
+        private readonly ReviewManager _reviewManager;
+        private const string ENDPOINT_SETTING = "Endpoint";
+        private readonly CommentsManager _commentsManager;
+        private readonly NotificationManager _notificationManager;
+        public readonly UserPreferenceCache _preferenceCache;
+
+        public string Endpoint { get; }
+        public ReviewModel Review { get; private set; }
+        public UsageSampleModel Sample { get; private set; }
+        public IEnumerable<UsageSampleModel> SampleRevisions { get; private set; }
+        public CodeLineModel[] SampleContent { get; set; }
+        public ReviewCommentsModel Comments { get; set; }
+        public string SampleOriginal { get; set; }
```

Comment:
What's the use of this field?

Reply:
Assuming this is referencing SampleOriginal, this field is used to allow editing of the sample markdown- it is kept separate so that the pop-up dialog can access it directly